internal/local/app.go

```diff
@@ -236,6 +238,33 @@ func (a *App) LoadConfig() error {
 	return nil
 }
 
+// ConfigContainsVersion returns true if the app root config file contains a version
+func (a *App) ConfigContainsVersion() bool {
+	switch a.Config {
+	case FileRealmConfig:
+		a.AppData = &AppRealmConfigJSON{}
+	case FileConfig:
+		a.AppData = &AppConfigJSON{}
+	case FileStitch:
+		a.AppData = &AppStitchJSON{}
+	}
+
+	path := filepath.Join(a.RootDir, a.Config.String())
+	file, openErr := os.Open(path)
+	if openErr != nil {
+		return false
+	}
+	defer file.Close()
+
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		if strings.Contains(scanner.Text(), a.AppData.ConfigVersion().String()) {
```

Comment:
I'm actually surprised this check is doing what we're expecting (do we have tests that prove this works as well as catches failures?)

basically, when you call `a.AppData.ConfigVersion()`, it's going to pull the data we've loaded into `AppData`, which doesn't seem to be anything _but_ the empty pointers youve assigned above in the switch/case.

what does `a.AppData.ConfigVersion().String()` look in each of these iterations?  we might just be checking something like: does the line contain a `"0"`

Reply:
Added some test cases that catch 'invalid' configs.